microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,297 @@
+      * Each `"..._api":` entry must have a single field, `"ServiceNotUsedForIdentifyingUserAcrossSites":`, with either a `true` or `false` value.  `true` means that this service is not used for identifying the user across sites.  `false` means that this service is used for identifying the user across sites.
+
+
+<!-- ====================================================================== -->
+## Deploy the Ad Selection API services
+
+The Ad Selection API uses a trusted execution environment (TEE) to provide a level of assurance for data integrity, data confidentiality, and code integrity; see [Confidential Computing: Hardware-Based Trusted Execution for Applications and Data](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/confidentialcomputing_outreach_whitepaper-8-5x11-1.pdf).  Services provided by the Ad Selection API must run in a TEE to secure the data used by these services.
+
+Ad Selection Services running in a TEE should be deployed on a cloud platform that supports the necessary security features.  Initially, services can be deployed in Azure using [Confidential ACI containers](/azure/container-instances/container-instances-confidential-overview).
+
+The Ad Selection API provides different services that need to be deployed by sellers or buyers.
+
+
+<!-- ------------------------------ -->
+#### Images for deploying services
+
+Add your User-Defined Functions within the provided images of services.  The User-Defined Functions can run custom business logic.  The User-Defined Functions run in private containers within the deployed services.
+
+Each image defines one service.  Some images don't contain UDFs, and consist entirely of predefined code.  Some images contain UDFs, as functions with an empty body or starter code, that are the relevant User-Defined Functions for that service.  Fill in the bodies of the User-Defined Functions in these images with your own custom code.
+
+Microsoft provides an image for each service, which can be deployed on a cloud provider.  These are the public images that must be used for deployment.  Only official images from Microsoft are able to run private auctions.
+
+Images for seller services:
+* [SellerFrontEnd service](https://mcr.microsoft.com/product/ad-selection/azure/seller-frontend-service)
+* [Auction service](https://mcr.microsoft.com/product/ad-selection/azure/auction-service)
+
+Images for buyer services:
+* [BuyerFrontEnd service](https://mcr.microsoft.com/product/ad-selection/azure/buyer-frontend-service)
+* [Bidding service](https://mcr.microsoft.com/product/ad-selection/azure/bidding-service)
+* [K-Anonymity service](https://mcr.microsoft.com/product/ad-selection/azure/k-anonymity-service)
+
+
+<!-- ------------------------------ -->
+#### Services for sellers
+
+| Service | Description |
+| --- | --- |
+| **SellerFrontEnd** | Provides a `/SelectAd` gRPC endpoint, which receives requests from the seller's ad service to initiate the Protected Audience auction flow. |
```

Comment:
Let's change this 

> Provides a `/SelectAd` HTTPS endpoint, which receives requests from the seller's untrusted ad service to initiate the Protected Audience auction flow.


And delete gRPC mentions in other places(3 places except this one), let's make just:

> Provides a `/GetBids` endpoint

Reply:
![image](https://github.com/user-attachments/assets/ea1c7fb3-76e0-49e2-b2a5-c3fbebb869d2)